padle/menu/padle_scroll.py

```diff
@@ -0,0 +1,80 @@
+from math import ceil
+from typing import Optional
+
+from discord import Message
+from discordmenu.embed.menu import EmbedControl, EmbedMenu
+from tsutils.menu.components.panes import MenuPanes
+
+from padle.view.padle_scroll_view import PADleScrollView, PADleScrollViewState
+
+
+class ScrollEmojis:
+    prev_page = '\N{BLACK LEFT-POINTING DOUBLE TRIANGLE}'
+    next_page = '\N{BLACK RIGHT-POINTING DOUBLE TRIANGLE}'
+    delete = '\N{CROSS MARK}'
```

Comment:
technically you do not need the `delete` here btw, the menu auto-does that for you, you can delete this line